v3/plugins/ms-vscode/vscode-github-pullrequest/0.20.0/meta.yaml

```diff
@@ -0,0 +1,15 @@
+publisher: ms-vscode
+apiVersion: v2
+name: vscode-github-pullrequest
+version: 0.20.0
+type: VS Code extension
+displayName: GitHub Pull Requests and issues
+title: Pull Request Provider for GitHub.
+description: This extension allows you to review and manage GitHub pull requests and issues.
+icon: https://raw.githubusercontent.com/microsoft/vscode-pull-request-github/0.20.0/resources/icons/github_logo.png
+repository: https://github.com/microsoft/vscode-pull-request-github
```

Comment:
Publish date should match the date (roughly) that the plugin was published in our registry

Reply:
Fixed